lib/link_preview/content.rb

```diff
@@ -337,6 +343,15 @@ def content_html_video
       EOF
     end
 
+    def content_html_iframe
+      return unless content_url.present?
+      width_attribute = %(width="#{content_width_scaled}") if content_width_scaled > 0
+      height_attribute = %(height="#{content_height_scaled}") if content_height_scaled > 0
+      <<-EOF.strip.gsub(/\s+/, ' ').gsub(/>\s+</, '><')
+          <iframe src="#{content_url}" #{width_attribute} #{height_attribute} />
```

Comment:
Not a big deal but the indentation here looks a little off (4 spaces instead of 2)